lib/adal.js

```diff
@@ -165,7 +165,9 @@ AuthenticationContext = function (config) {
  * Gets initial Idtoken for the app backend
  * Saves the resulting Idtoken in localStorage.
  */
-AuthenticationContext.prototype.login = function () {
+AuthenticationContext.prototype.login = function (loginReason) {
+    loginReason = loginReason || '';
```

Comment:
yes it can be null coming in, but if it is shouldn't we provide an empty string for telemetry purposes?